src/staff_rename.lua

```diff
@@ -317,15 +484,18 @@ function staff_rename()
 
                 str.LuaString = abb_fonts[i]..str.LuaString
                 staff:SaveNewAbbreviatedNameString(str)
+                if autonumber_check[i]:GetCheck() == 1 then
+                    staff.UseAutoNumberingStyle = true
+                else
+                    staff.UseAutoNumberingStyle = false
+                end
+                staff.AutoNumberingStyle = autonumber_popup[i]:GetSelectedItem()
                 staff:Save()
                 ::done2::
             end
         end
-
     end -- function
-
     dialog("Rename Staves")
-
 end -- rename_staves()
 
 staff_rename()
```

Comment:
Nit: can you add a newline at the end of the file? See comment below on why.

Reply:
Like hit return one more time?